pkg/kotsclient/cluster_list.go

```diff
@@ -5,22 +5,39 @@ import (
 	"net/http"
 	"strconv"
 
+	"github.com/pkg/errors"
+
 	"github.com/replicatedhq/replicated/pkg/types"
 )
 
 type ListClustersRequest struct {
 }
 
 type ListClustersResponse struct {
-	Clusters []*types.Cluster `json:"clusters"`
+	Clusters      []*types.Cluster `json:"clusters"`
+	TotalClusters int              `json:"totalClusters"`
 }
 
-func (c *VendorV3Client) ListClusters(includeTerminated bool) ([]*types.Cluster, error) {
+func (c *VendorV3Client) ListClusters(includeTerminated bool, startTime string, endTime string) ([]*types.Cluster, error) {
```

Comment:
`startTime` and `endTime` should be *time.Time not string.  The caller should validate the input and display an appropriate message if the format is not correct.